### ISSUE TITLE
How to use in regular .js file?

### ISSUE THREAD

**Issue by jimmiejackson414** [microcipcip/cookie-universal#93]
I saw your wonderful response to #87 and it very nearly fixed what I'm looking to do (I went with your option number 2), however I've run into a slightly different scenario.

 I created a plugin called `cookies.js` which I am setting in my nuxt.config.js. I've also got a file called `~/helpers/requestHeaders.js`. All it does is return the token that is stored in a cookie, and includes it in a request. They look like this:

**cookies.js:**
```
import cookieUniversal from 'cookie-universal';

export default ({ req, res }, inject) => {
  const options = { alias: 'cookies', parseJson: true };
  inject(options.alias, cookieUniversal(req, res, options.parseJson));
};

```

**requestHeaders.js:**
```
import cookieUniversal from 'cookie-universal';

const options = { parseJson: true };
const Cookies = cookieUniversal(false, false, options.parseJson);

function requestHeaders () {
  return { authorization: `Bearer ${Cookies.get('gc-token')}` };
};

export default requestHeaders();
```

requestHeaders doesn't work on the server side however. You mention at the end of your reply that those functions won't work because we won't have access to the `req` and `res` objects. 

Is there a way - with either this library or `cookie-universal-nuxt` - to access cookies from any file on both server and client?

**Comment by microcipcip** [comment]
@jimmiejackson414 sorry I missed your message. You have to pass `req` and `res` to the function to be able to use the library. There is no other way. Look at the usage in the demo I have created for [express ](https://github.com/microcipcip/cookie-universal/blob/master/packages/cookie-universal/demo/server-express.js#L7)and for a pure [node server](https://github.com/microcipcip/cookie-universal/blob/master/packages/cookie-universal/demo/server.js#L18). I think you need a function like this:
```
...
export default function requestHeaders (req, res) {
  const Cookies = cookieUniversal(req, res, options.parseJson);
  return { authorization: `Bearer ${Cookies.get('gc-token')}` };
};
```

**Comment by microcipcip** [comment]
In express we have a code like this:
```
var express = require('express')
var cookieParser = require('cookie-parser')

var app = express()
app.use(cookieParser())
```
The `cookieParser` function if expanded looks something like this:
```
function cookieParser(req, res, next) {
  // Code for injecting 'cookies' globally here
  next();
}
```
So that at the end you get something like this:
```
app.use(function(req, res, next) {
  // Code for injecting 'cookies' globally here
  next();
});
```

**Comment by jimmiejackson414** [comment]
Ah that makes perfect sense, thank you!

I figured out another way to handle it as well:
In the created() hook of my `default.vue` layout, I added this:
```
created () {
  const token = this.$cookies.get('gc_token');
  this.$graphql.setHeaders({ authorization: `Bearer ${token}` });
},
```
`this.$graphql` refers to the nuxt-graphql-request library I am using, so that when the layout is created the token is set for all requests. I think I'll need to incorporate something like what you've shown above however for certain other cases, so thank you for following up with this!
